android/src/main/kotlin/com/jhomlala/better_player/BetterPlayerPlugin.kt

```diff
@@ -708,12 +786,20 @@ class BetterPlayerPlugin : FlutterPlugin, ActivityAware, MethodCallHandler {
         private const val STOP_PRE_CACHE_METHOD = "stopPreCache"
         private val PIP_ASPECT_RATIO = Rational(16, 9)
 
-        /** For custom action in PIP mode  */
-        private const val DW_NFC_BETTER_PLAYER_CUSTOM_PIP_ACTION = "better_player.nfc_ch_app/pip_custom_action"
-        private const val EXTRA_ACTION_TYPE = "extra_action_type"
-        private enum class PipActions(val rawValue: Int) {
+        /** For custom action from outside the app */
+        const val DW_NFC_BETTER_PLAYER_CUSTOM_ACTION =
+            "better_player.nfc_ch_app/custom_action"
+        const val EXTRA_ACTION_TYPE = "extra_action_type"
+
+        enum class CustomActions(val rawValue: Int) {
             PLAY(1),
             PAUSE(2)
         }
+
+        // Will be observed to show notification.
+        var notificationParameter: MutableLiveData<NotificationParameter?> = MutableLiveData()
+        // Will be observed to update action in notification.
+        var notificationActions: MutableLiveData<List<NotificationCompat.Action>?> =
+            MutableLiveData()
```

Comment:
I think better to use LiveData for non private parameter. But could not find the way this time.